lib/scope_file_access_flat.rb

```diff
@@ -30,7 +30,11 @@ def binary?(system_file)
     path = system_file.scope.file_path(system_file)
     return false if File.zero?(path)
 
-    output = Cheetah.run("file", path, stdout: :capture)
-    !output.include?("ASCII")
+    # Code by https://github.com/djberg96/ptools
+    bytes = File.stat(path).blksize
+    bytes = 4096 if bytes > 4096
+    s = (File.read(path, bytes) || "")
+    s = s.encode('US-ASCII', :undef => :replace).split(//)
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.